docs/pages/admin-guide.mdx

```diff
@@ -1153,6 +1153,22 @@ spec:
       kubernetes_groups: ["system:masters"]
 ```
 
+Assigning multiple teams within a single organization is as follows:
+
+```yaml
+   teams_to_logins:
+      - organization: octocats
+        team: team-1
```

Comment:
Looks good. May be have a different target role set for two different teams to highlight the difference?

Reply:
Makes sense. I updated the example to assign one team to the admin role, and another to the developers role